package-helpers.psm1

```diff
@@ -86,41 +50,40 @@ function Invoke-Promote {
         # View to promote into
         [Parameter(Mandatory = $true)]
         [String]
-        $View
+        $ViewId,
+
+        # Git ref (short) for the release tag ex: v1.3.5
+        [Parameter(Mandatory = $true)]
+        $ReleaseRef
     )
 
 
-    $packages = @("EdFi.DataManagement.Service")
+    $package = "EdFi.DataManagementService"
+    $version = $ReleaseRef -replace "v", ""
 
     $body = @{
         data      = @{
-            viewId = $View
+            viewId = $ViewId
         }
         operation = 0
-        packages  = $packages
-    }
-
-    $latestPackages = Get-PackagesFromAzure -Packages $packages
-
-    foreach ($key in $latestPackages.Keys) {
-        $body.packages += @{
-            id           = $key
-            version      = $latestPackages[$key]
-            protocolType = "NuGet"
-        }
-    }
```

Comment:
The older code tried to look up all of the available versions. That was too aggressive. Besides, we know exactly which version we want to promote in Azure Artifacts.